tools/yapf.bat

```diff
@@ -0,0 +1,16 @@
+@ECHO off
+REM Copyright 2013 The Flutter Authors. All rights reserved.
+REM Use of this source code is governed by a BSD-style license that can be
+REM found in the LICENSE file.
+
+REM ---------------------------------- NOTE ----------------------------------
+REM
+REM Please keep the logic in this file consistent with the logic in the
```

Comment:
You might want to add this same comment to yapf.bat.

Reply:
Maybe you mean yapf.sh? Anyway I added the note to that file as well, makes sense to keep this consistent.

Reply:
Yes, that's what I meant.